tests/Unit/Domain/Test_Neighbors.cpp

```diff
@@ -0,0 +1,80 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <catch.hpp>
+
+#include "Domain/Neighbors.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+TEST_CASE("Unit.Domain.Neighbors", "[Domain][Unit]") {
+  // Test default constructor:
```

Comment:
Add same comment as for BlockNeighbors about not testing values because of serialization